src/libs/ajax/DataRepo.ts

```diff
@@ -48,6 +48,7 @@ export type SnapshotBuilderFeatureValueGroup = {
 
 export type SnapshotBuilderDatasetConceptSets = {
   name: string;
+  concept: SnapshotBuilderConcept;
```

Comment:
```suggestion
  concept?: SnapshotBuilderConcept;
```

Reply:
Is it true that only a domain concept set will have an associated concept and the prepackaged concept sets will not?